src/core_ocean/mode_forward/mpas_ocn_forward_mode.F

```diff
@@ -597,9 +595,13 @@ function ocn_forward_mode_run(domain) result(ierr)!{{{
            call mpas_timer_stop("land_ice_build_arrays")
 
            call ocn_frazil_forcing_build_arrays(domain, meshPool, forcingPool, diagnosticsPool, statePool, err)
-           
            call ocn_tidal_forcing_build_array(domain, meshPool, forcingPool, diagnosticsPool, statePool, err)
 
+           ! Compute normalGMBolusVelocity, relativeSlope and RediDiffVertCoef if respective flags are turned on
+           if (config_use_Redi.or.config_use_GM) then
+               call ocn_gm_compute_Bolus_velocity(statePool, diagnosticsPool, meshPool, scratchPool, timeLevelIn=1)
```

Comment:
Looks like the stand-alone call is here, so maybe this is just a matter of removing the commented out lines for clarity.